chirp/drivers/anytone.py

```diff
@@ -685,9 +685,10 @@ def filter(s):
                 s_ += (c if c in chirp_common.CHARSET_ASCII else "")
             return s_
 
+        welcome_msg = ''.join(list(filter(_settings.welcome)))
```

Comment:
Why the extra list()? I don't think it should be necessary right?

Reply:
Right!